src/Configuration/CFGCommon/CFGArg.json

```diff
@@ -251,4 +251,4 @@
         "  programmer list_cable"
     ]}
   ]
-}
\ No newline at end of file
+}
```

Comment:
Is this adding whitespace or "}" ?

Reply:
it's new line char. QtCreator puts new line char at the end of file by default.